src/parameters.rs

```diff
@@ -34,9 +34,12 @@ use crate::errors::CredentialError;
 pub(crate) fn sizeof_system_parameters(number_of_attributes: u8) -> usize {
     // G_y is always at least three elements
     if number_of_attributes < 3 {
-        return 32 * (5 + 3 + number_of_attributes as usize + 4) + 1
+        // 32 * (5 + 3 + number_of_attributes as usize + 4) + 1
+        385 + 32 * number_of_attributes as usize
+    } else {
+        // 32 * (5 + (2 * number_of_attributes as usize) + 4) + 1
+        289 + 64 * number_of_attributes as usize
     }
-    32 * (5 + (2 * number_of_attributes as usize) + 4) + 1
 }
```

Comment:
I expected this to be something like `289 + 64 * core::cmp::max(3, number_of_attributes as usize)`, why isn't it?